libs/recnet-api-model/src/lib/api/article.ts

```diff
@@ -5,10 +5,29 @@ import { articleSchema } from "../model";
 // GET /articles
 export const getArticlesParamsSchema = z.object({
   link: z.string().url(),
+  useDigitalLibrary: z.union([
+    z.boolean(),
+    z.enum(["true", "false"]).transform(val => val === "true")
+  ]).default(true)
 });
 export type GetArticlesParams = z.infer<typeof getArticlesParamsSchema>;
 
 export const getArticlesResponseSchema = z.object({
   article: articleSchema.nullable(),
 });
 export type GetArticlesResponse = z.infer<typeof getArticlesResponseSchema>;
+
+
+// PATCH /articles/admin
+export const AdminUpdateArticleDtoSchema = z.object({
```

Comment:
We have a certain naming convention for API schema. For example, this API is `PATCH /articles/admin`, we will name this schema `patchArticlesAdminRequestSchema`.